src/window.rs

```diff
@@ -682,6 +682,23 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused. Providing `None` will unset the request for user attention.
+    ///
+    /// Unsetting the request for user attention is automatically done by the WM when
+    /// the window receives focus.
```

Comment:
The specification is not able to guarantee this, so we should not do so either. If we state anything, we should state the opposite.

Reply:
Both Windows and macOS do give this guarantee though only X11 doesn't. That's why I put a platform-specific comment on X11 "warning" the user we can't give this guarantee.

Reply:
Pointing out that it isn't done on all platforms is much more useful though. It tells you that it might be done on some, but can't be taken for granted.